src/avatar.scss

```diff
@@ -97,17 +97,21 @@ $block: #{$fd-namespace}-avatar;
 
   // placeholder background
   &--placeholder {
-    background-color: var(--sapContent_ImagePlaceholderBackground);
+    background-color: var(--fdAvatar_Placeholder_BackgroundColor);
+    border: var(--fdAvatar_Border);
+    color: var(--fdAvatar_Placeholder_DefaultColor);
   }
 
   // tile background
   &--tile {
-    background-color: var(--sapTile_IconColor);
+    background-color: var(--fdAvatar_Tile_BackgroundColor);
+    border: var(--fdAvatar_Border);
+    color: var(--fdAvatar_Tile_DefaultColor);
   }
 
   // optional border
   &--border {
-    border: 0.0625rem solid var(--sapGroup_ContentBorderColor);
+    border: var(--fdAvatar_Optional_Border);
```

Comment:
This one can just be `0.0625rem solid var(--sapGroup_ContentBorderColor);` because it's the same in all themes. We can delete `--fdAvatar_Optional_Border`